content/ops/contingency-plan.md

```diff
@@ -0,0 +1,122 @@
+  * We need access to our code and documentation, which means access to the information in our 18F GitHub organization. If we lose access to GitHub, each cloud.gov team member has local copies of some cloud.gov git repositories created as part of their ordinary day-to-day work, which can serve as a temporary resource. The 18F Director of Infrastructure makes systematic backups of 18F GitHub repositories at a daily frequency.
+* Our systems are secure (we maintain confidentiality and integrity). Plan to restore this function if disrupted:
+  * If this is threatened, we follow our [Security Incident Response Guide](hhttps://docs.cloud.gov/ops/security-ir).
+* The team can communicate with customers about the status of cloud.gov. Key components and plans to restore this function if disrupted:
+  * If we have more than a small amount of service disruption, team leads will post to our <span class="c10 c6">[externally-hosted status page](https://cloudgov.statuspage.io), post to the 18F Slack #cloud-gov-support channel, and email key contacts at customer teams. If the status page and/or Slack are unavailable, we’ll use GSA email&nbsp;as a backup method to communicate with our key customers. If that’s not possible, we can fall back to phone numbers.
+  * If the disruption is significantly affecting the delivery of our services to customers, team leads will coordinate with the 18F commsquad on how we should communicate this externally, including potentially using @18F Twitter and later following up with an 18F blog post.
+* Our core services and customer services are online and functional (we maintain availability). The team should be able to fix this (and the following items on this list) if all of the above are functioning.
+  *  cloud.gov supports customers doing self-service modifications and updates to their systems, such as the web user interface and command line login.
+  *  All of our systems are online and functional (full cloud.gov website, etc.).
+
+---
+
+## Contingency roles, responsibilities, and contact information
+
+Reference: [team roster sheet](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit) with names and ordinary team roles.
+
+Roles we probably need to define for this plan (based on [https://github.com/18F/compliance-docs/blob/master/CP-Policy.md#4-roles-and-responsibilities](https://github.com/18F/compliance-docs/blob/master/CP-Policy.md)):
+
+* Development Operations
+* Disaster Recovery Team
+* Senior Director
+* Director of Compliance
+
+People who need to get notified when there's a serious problem:
+
+If this is not a security incident and just unplanned downtime:
+
+* cloud.gov team
+* Infrastructure and @commsquad
+* Key 18F staff (comms lead, director of infrastructure, etc.)
+* cloud.gov users (through statuspage)
+* Designated FedRAMP personnel (when applicable)
```

Comment:
We still need to figure it out but it is going to be part of the FedRAMP JAB process. They will give us some contacts to notify when 💩  hits the fan.